checkov/kubernetes/checks/resource/k8s/ApiServerEtcdCaFile.py

```diff
@@ -8,15 +8,15 @@
 class ApiServerEtcdCaFile(BaseK8sContainerCheck):
     def __init__(self) -> None:
         id = "CKV_K8S_102"
-        name = "Ensure that the --etcd-ca-file argument is set as appropriate"
+        name = "Ensure that the --etcd-cafile argument is set as appropriate"
         super().__init__(name=name, id=id)
 
     def scan_container_conf(self, metadata: Dict[str, Any], conf: Dict[str, Any]) -> CheckResult:
         self.evaluated_container_keys = ["command"]
         keys, values = extract_commands(conf)
 
         if "kube-apiserver" in keys:
-            if "--etcd-ca-file" not in keys:
+            if "--etcd-cafile" not in keys:
```

Comment:
for some reason your old changes are still in this PR, can you please remove them?

Reply:
I'm not sure if I can remove them I will have to raise the PR again. I shall do that!